src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6653,7 +6653,7 @@ private bool IsUsingAliasInScope(string name)
             var lookupResult = LookupResult.GetInstance();
             try
             {
-                LookupOptions options = LookupOptions.AllMethodsOnArityZero;
+                LookupOptions options = LookupOptions.AllMethodsOnArityZero | LookupOptions.SearchInExtensionTypes;
```

Comment:
>LookupOptions options = LookupOptions.AllMethodsOnArityZero | LookupOptions.SearchInExtensionTypes; [](http://example.com/codeflow?start=16&length=99)

It looks like I am missing the place where we look for extension members after overload resolution fails.

Reply:
You're correct. I need to add a section to the spec for "method invocations", where we only move forward to extension methods (and extension types) if overload resolution failed.
[https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#12892-method-invocations](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#12892-method-invocations)